ext/opcache/ZendAccelerator.c

```diff
@@ -840,7 +840,7 @@ static inline int accel_is_inactive(void)
 	if (ZCG(accel_directives).force_restart_timeout
 		&& ZCSG(force_restart_time)
 		&& time(NULL) >= ZCSG(force_restart_time)) {
-		zend_accel_error(ACCEL_LOG_WARNING, "Forced restart at %ld (after " ZEND_LONG_FMT " seconds), locked by %d", time(NULL), ZCG(accel_directives).force_restart_timeout, mem_usage_check.l_pid);
+		zend_accel_error(ACCEL_LOG_WARNING, "Forced restart at %ld (after " ZEND_LONG_FMT " seconds), locked by %d", (long)time(NULL), (zend_long)ZCG(accel_directives).force_restart_timeout, mem_usage_check.l_pid);
```

Comment:
The `(zend_long)` cast here is not necessary, force_restart_timeout is already zend_long.

Reply:
It was for systems like OpenBSD where time_t are 64 bits types.

Reply:
The (long)time(NULL) cast makes sense, I mean the (zend_long) cast on `ZCG(accel_directives).force_restart_timeout`, which is already zend_long.